src/DIRAC/DataManagementSystem/DB/FileCatalogComponents/DirectoryMetadata/MultiVODirectoryMetadata.py

```diff
@@ -40,7 +41,10 @@ def _getMetaNameDict(metaDict, credDict):
 
     fMetaDict = {}
     for meta, value in metaDict.items():
-        fMetaDict[_getMetaName(meta, credDict)] = value
+        if meta in MetaQuery.FILE_STANDARD_METAKEYS:
+            fMetaDict[meta] = value
+        else:
+            fMetaDict[_getMetaName(meta, credDict)] = value
```

Comment:
`_getMetaName` is used in 10 places (according to github code reference finder). Is this the only place where this makes a difference? Why not implement this check for standard keys in `_getMetaName`?

Reply:
A good point. This is what we found when trying to select files based on creation date in a multi VO environment. The build-in meta keys are not VO aware, so there should be no VO suffix used when searching. The method in question is is here:
[https://github.com/DIRACGrid/DIRAC/blob/6764f043334b7d90f28df8fa7b62e41e1cb835c9/src/DIRAC/DataManagementSystem/DB/FileCatalogComponents/DirectoryMetadata/MultiVODirectoryMetadata.py#L183
](https://github.com/DIRACGrid/DIRAC/blob/6764f043334b7d90f28df8fa7b62e41e1cb835c9/src/DIRAC/DataManagementSystem/DB/FileCatalogComponents/DirectoryMetadata/MultiVODirectoryMetadata.py#L183)

There are 4 `_getMetaNameDict` calls, 2 for files and 2 for directories. 

As you said there are multiple calls to `_getMetaName`, adding a modification there would be most like not harmful, Let me run some checks to see if its OK. Many of those calls are used for user metadata and are kept in dedicated DB tables (with a VO suffix).

Reply:
Done.